layouts/partials/pagelayout.html

```diff
@@ -0,0 +1,33 @@
+{{ $section_to_display := $.Scratch.Get "section_to_display"  }}
+<div class="w-100 ph4 pb5 pb6-ns pt1 mt4 pt3-ns">
+  <div class="flex">
+    <div class="w-100 w-80-l ph0">
+      <article class="w-100 nested-copy-line-height nested-links nested-img pv4">
+        <h1 class="primary-color-dark f2">
+          {{ if .IsHome -}}
+            Hugo Themes
+          {{ else }}
+            Hugo Themes <span class="f4 mt2 mt0-l db dib-l">({{ with $.Data.Singular }}{{ . | humanize }}:{{ end }} {{ .Title }})</span>
+
+          {{- end }}
+        </h1>
+        <div class="{{ .Site.Params.copyClass }} mid-gray f5 f4-l">
+          {{ .Content }}
+        </div>
+      </article>
+      <div class="flex flex-wrap justify-left pr4">
+        {{ $globalVariables := .Site.Data.globals }}
```

Comment:
Everything starting with "$" is by definition a variable, so no need to say it again, "$globals" is better.